renovate.yaml

```diff
@@ -1,10 +1,10 @@
 package:
   name: renovate
   version: 37.144.0
-  epoch: 0
+  epoch: 1
   description: "Automated dependency updates. Multi-platform and multi-language."
   copyright:
-    - license: AGPL-3.0
+    - license: AGPL-3.0-or-later
```

Comment:
renovate has  it as AGPL-3.0-only on the README https://github.com/renovatebot/renovate